### PR TITLE
Added Action create methods for Collections and Thngs

### PR DESCRIPTION
**Pull request by ramonPaulAlvarez**

### CODE REVIEW

File: src/evrythng/entities/collections_.py [hunk 1]
```diff
@@ -19,6 +19,23 @@
...
+def create_collection_action(type_, collection=None, timestamp=None, identifiers=None,
+                             location=None, locationSource=None, context=None, customFields=None,
+                             api_key=None):
+    """Create an Action for a Collection"""
+    kwargs = locals()
+    kwargs['type'] = kwargs['type_']
+    del kwargs['type_']
+    api_key = kwargs.pop('api_key', None)
+
+    from evrythng.entities.actions import field_specs as action_field_specs
```
Comment (ramonPaulAlvarez) [inline src/evrythng/entities/collections_.py]: Is this ok?  Seemed preferable to managing additional field_specs for Actions and Thngs.

Comment (codeadict) [reply src/evrythng/entities/collections_.py]: I'm not a big fan of all those named arguments with None, i'd rather provide `type` and `collection` as a required argument and other stuff as a dict of **kwargs.

Comment (LyleScott) [reply src/evrythng/entities/collections_.py]: Do these imports need to be local?

File: src/evrythng/entities/thngs.py [hunk 2]
```diff
@@ -24,6 +24,22 @@
 }
 
 
+def create_thng_action(type_, thng=None, timestamp=None, identifiers=None, location=None,
```
Comment (codeadict) [inline src/evrythng/entities/thngs.py]: Same thing.

**Review by LyleScott — COMMENTED** [review]
Seems OK. Just wondering why the local import

File: src/evrythng/entities/collections_.py (hunk 1 above)
Comment (ramonPaulAlvarez) [reply src/evrythng/entities/collections_.py]: Model habit, moving them.

**Review by codeadict — COMMENTED** [review]
It should work, just left a concern about the function arguments that doesn't seem too elegant from my pov

File: src/evrythng/entities/collections_.py (hunk 1 above)
Comment (ramonPaulAlvarez) [reply src/evrythng/entities/collections_.py]: Good call on the required fields.  The arguments were duplicated from the original create_action function.  I'll leave the optional arguments as is for now though.

**Pull request merged**